Dockerfile

```diff
@@ -34,14 +34,28 @@ RUN --mount=target=. \
     -ldflags='-w -s' \
     -o /go/bin/goshimmer
 
+# Docker cache will be invalidated for RUNs after ARG definition (https://docs.docker.com/engine/reference/builder/#impact-on-build-caching)
+ARG CUSTOM_SNAPSHOT
```

Comment:
Can we rename this to CUSTOM_SNAPSHOT_URL?

Reply:
Sure!